src/qt/textctrl.cpp

```diff
@@ -61,6 +69,36 @@ void wxQtLineEdit::returnPressed()
     }
 }
 
+#if QT_VERSION >= QT_VERSION_CHECK( 5, 12, 0)
+void wxQtLineEdit::inputRejected()
+{
+    wxTextCtrl *handler = GetHandler();
+    if( handler )
+    {
+        QLineEdit *line_edit = ((QLineEdit *) handler->GetHandle());
```

Comment:
Is this different from what `GetEditable()` returns? If so, a comment explaining why/how would be useful. If not, why not just use it here?